src/early_multicellular_stage/editor/CellBodyPlanEditorComponent.cs

```diff
@@ -96,6 +99,9 @@ public partial class CellBodyPlanEditorComponent :
     private PackedScene microbeScene = null!;
 
     private CellPopupMenu cellPopupMenu = null!;
+
+    private ReproductionOrderEditor<CellTemplate, EarlyMulticellularEditor, CombinedEditorAction, EditorAction,
```

Comment:
Shouldn't this use the actual type used by the scene (`EarlyMulticellularReproductionOrderEditor`)? That'd simplify the long list of generic parameters defined here.